microsoft-edge/devtools-guide-chromium/whats-new/2022/04/devtools-101.md

```diff
@@ -0,0 +1,128 @@
+
+When you first load the webpage, the **Source Maps Monitor** tool shows that the source maps aren't cached yet:
+
+![Source Maps Monitor tool, upon first load.](devtools-101-images/source-maps-indexeddb-first-load.png)
+
+When you reload the webpage, the **Source Maps Monitor** tool shows that the cached source maps are used:
+
+![Source Maps Monitor tool, upon second load.](devtools-101-images/source-maps-indexeddb-second-load.png)
+
+
+<!-- ====================================================================== -->
+## Tracking pending network requests when exporting HAR files
+
+<!-- Title: Exporting HAR files from the Network tool now includes pending requests -->
+<!-- Subtitle: Use the new "Status text" column and "Include pending requests in HAR files" option in the Network tool. -->
+
+In Microsoft Edge 101, the new **Status text** column has been added to the **Network** tool.  The **Status text** column provides human-readable equivalent messages for the [HTTP status codes](https://developer.mozilla.org/docs/Web/HTTP/Status) that are reported in the **Status** column.
+
+In addition, a new setting has been added to the Network tool: **Include pending requests in HAR file**.  If you need to record **Pending** network requests when exporting a HAR file in the Network tool, turn on the **Include pending requests in HAR file** setting.  The **Status text** column will now include the **(pending)** state for any pending network requests.
+
+![The "Status text" column in the Network tool.](devtools-101-images/status-text-column-network-tool.png)
+
+To learn more about the available columns in the request log, see [Network features reference > Add or remove columns](../../../network/reference.md#add-or-remove-columns).
+
+
+<!-- ====================================================================== -->
+## Updated UI and camera controls in 3D View
+
+<!-- Title: Improvements to the 3D View tool -->
+<!-- Subtitle: Check out 3D View for updates to the UI and smoother camera controls. -->
+
+In Microsoft Edge 101, the **3D View** tool has two new buttons: **Pan view** (![The 'Pan view' icon.](devtools-101-images/pan-view-icon.png)) and **Rotate view** (![The 'Rotate view' icon.](devtools-101-images/rotate-view-icon.png)).  Use these buttons to switch between panning around the canvas or rotating the canvas.  Click the **Pan view** or **Rotate view** button, and then drag in the desired direction with your mouse, or use the arrow keys.
+
+The UI now provides a more uniform and intuitive experience.  The panes are organized more clearly, and accessibility issues have been fixed.
+
+![Updated UI and camera controls in 3D View.](devtools-101-images/3d-view-ui-camera-ctrls.png)
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../../../3d-view/index.md).
```

Comment:
Will this document be updated by the time this comes out?

Reply:
updated 3D View article.